termwiz/src/render/terminfo.rs

```diff
@@ -47,6 +47,7 @@ impl TerminfoRenderer {
         });
     }
 
+    #[cfg_attr(feature = "cargo-clippy", allow(clippy::cognitive_complexity))]
```

Comment:
ah well, it was good while it lasted :)

Reply:
Apparently 27 is too much.  Need `cargo clippy --i-am-a-genius`.